src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/VisualBasicDefinitionMap.vb

```diff
@@ -7,6 +7,7 @@ Imports Microsoft.CodeAnalysis.CodeGen
 Imports Microsoft.CodeAnalysis.Emit
 Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
 Imports Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
+Imports Microsoft.CodeAnalysis.Symbols
```

Comment:
>Microsoft.CodeAnalysis.Symbols [](start = 8, length = 30)

Please sort or remove if not needed.

Reply:
Hmm, not sure why this was added everywhere...

---
In reply to: [302288682](https://github.com/dotnet/roslyn/pull/36854#discussion_r302288682) [](ancestors = 302288682)